test/widgets/settings_test.dart

```diff
@@ -64,6 +64,24 @@ void main() {
         hasCheckedState: true, isChecked: expectedIsChecked);
   }
 
+  group('SettingsPage scroll behavior', () {
+    testWidgets('content is scrollable when taller than a screenful', (tester) async {
```

Comment:
nit: can simplify by not wrapping this test in a group (since it was the only thing there anyway):
```suggestion
  testWidgets('SettingsPage content is scrollable when taller than a screenful', (tester) async {
```

Reply:
bump — I do think the "SettingsPage" part is helpful, for setting context when this test case goes by in `flutter test` output